plugins/module_utils/rds.py

```diff
@@ -35,6 +35,8 @@
     "remove_tags_from_resource",
     "list_tags_for_resource",
     "promote_read_replica_db_cluster",
+    "stop_db_cluster",
+    "start_db_cluster",
```

Comment:
Can you please cover these two with units tests? https://github.com/ansible-collections/amazon.aws/blob/main/tests/unit/module_utils/test_rds.py